src/lib/holocene/alert.svelte

```diff
@@ -1,18 +1,26 @@
 <script lang="ts">
   import Icon from '$lib/holocene/icon/icon.svelte';
+  import type { HTMLAttributes } from 'svelte/elements';
   import type { IconName } from './icon/paths';
 
+  interface $$Props extends HTMLAttributes<HTMLDivElement> {
+    intent: 'warning' | 'error' | 'success' | 'info';
+    title: string;
+    icon?: IconName;
+    bold?: boolean;
+    'data-testid'?: string;
+  }
+
   export let intent: 'warning' | 'error' | 'success' | 'info';
   export let title: string;
   export let icon: IconName = null;
   export let bold = false;
+
+  let className = '';
+  export { className as class };
 </script>
 
-<div
-  class="alert {intent} {$$props.class}"
-  class:bold
-  data-testid={$$props.testId}
->
+<div class="alert {intent} {className}" class:bold {...$$restProps}>
```

Comment:
Should we just add `role="alert"` here so we don't even have to think about adding it when we use an Alert?

Reply:
Or at least make it a required prop?

Reply:
I was wondering about that... Most `Alert`s will need a role, but there are are cases in the UI currently where the `Alert` holds information that doesn't change and therefore isn't really an `alert` or a `status`. I opted to make `role` optional because of those cases.

Maybe we want to create another component not called `Alert` for those cases though? Called `Banner` or something? Or maybe we leave it as is for now and iterate on it in the future 😅

Reply:
Hm I see. Yeah sounds like things that we are currently using an Alert for should be maybe not be Alerts. I just would hate to have to keep going back and fixing a11y errors, ideally in using the holocene components we get good a11y baked in.

Reply:
> ideally in using the holocene components we get good a11y baked in

100% agree. I'm going to update `Alert` to default to `role="alert"` and set `role="status"` on ones that don't need it.